packages/start-server-core/src/createStartHandler.ts

```diff
@@ -49,8 +49,10 @@ export function getStartResponseHeaders(opts: { router: AnyRouter }) {
 
 export function createStartHandler<TRouter extends AnyRouter>({
   createRouter,
+  RootDocument,
 }: {
   createRouter: () => TRouter
+  RootDocument?: any
```

Comment:
i wasn't sure how to type this since its a Solid component but its in the core package